### DISCUSSION TITLE
Alias Padding

### DISCUSSION THREAD

**Discussion by matthewshammond** [FelixKratz/SketchyBar#232] [Q&A] [answered]
Is there a way to change the left and right padding for aliases?
I am using aliases for the right side for clock and battery and have:
```
sketchybar --add alias "Control Center,Clock" right                            \
           --set "Control Center,Clock" alias.color=$FROST4                    \
                                                                               \
           --add alias "Control Center,Battery" right                          \
           --set "Control Center,Battery" script="$PLUGIN_DIR/battery.zsh" 
```
However, I can't adjust the spacing between.  I've tried doing the same as icons/labels/backgrounds, but have not been able to figure out a way.

Also when I use `sketchybar --query default_menu_items` there are two `"Control Center,Clock",` which one is Day and Time and the other is Date....is it possible to get Day Date Time like on the Mac toolbar?

Thank you and appreciate your work on this project.

**Comment by FelixKratz** [comment, marked as answer]
Aliases have all regular item properties available, so:
```bash
sketchybar --set "Control Center,Battery" background.padding_left=10 background.padding_right=10
```
should already do the trick. Paddings can also be negative.

If there are two menubar components with the same name, we can not alias them consistently. I would propose the use a clock plugin as it is displayed in the default config.
